test/common/protobuf/utility_test.cc

```diff
@@ -45,7 +46,15 @@ using namespace std::chrono_literals;
 
 namespace Envoy {
 
-using testing::HasSubstr;
+using ::testing::HasSubstr;
+
+template <class M> bool checkProtoEquality(M proto1, std::string text_proto2) {
```

Comment:
I think this can be a non-templated function, because all its uses are where the first param is of type `ProtobufWkt::Value`.
Leaving it up to you to decide whether it needs to be concrete or not.

Reply:
Done